vcx/ci/Jenkinsfile

```diff
@@ -10,9 +10,9 @@ testing()
 def testing() {
     stage('Testing') {
         parallel([
-            'Main' : { mainUbuntu() },
+            // 'Main' : { mainUbuntu() },
             'Android': { android() },
-            'iOS' : { ios() }
+            // 'iOS' : { ios() }
         ])
```

Comment:
Naga, do these need to be uncommented so that the Main and iOS builds will still proceed.